py_zipkin/logging_helper.py

```diff
@@ -87,11 +89,23 @@ def log_spans(self):
         a success. It also logs the service (`ss` and `sr`) or the client
         ('cs' and 'cr') annotations.
         """
+
+        # FIXME: Should have a single aggregate handler
+        if self.firehose_handler:
+            self._log_spans_with_span_sender(
+                ZipkinBatchSender(self.firehose_handler,
+                                  self.max_span_batch_size)
```

Comment:
the 2 max_batch_size don't have to be the same as the zipkin api and collectors don't have the same MTU limitations we have for firehose

Reply:
I added this as a FIXME for this branch, since I am not sure how we want to handle this, so for now we can assume both max sizes are the same.